docs/Catalog/VisualizationDashboards.md

```diff
@@ -114,10 +114,8 @@ bucket contents and displays thumbnail image previews in a
 paginated grid (25 per page by default) of 
 all [supported image types](../Catalog/Preview.md#binary-and-special-file-format-previews).
 
-> To hide this block, specify the `noOverviewImages` flag in 
```

Comment:
should we deprecate `noOverviewImages` then?

Reply:
Yes, I think so
Maybe in a separate PR, so it would have a dedicated Changelog entry and would be noticed.

Reply:
I don't know if deprecation deserves mention in docs.
Probably, we can use "deprecated" keyword in Schema, and show `console.warn` on Overview page if `noOverviewImages` is in use.

Reply:
Added task to Asana. Also, for `noDownload` flag as well